package.json

```diff
@@ -51,19 +52,26 @@
     "eslint-plugin-import": "2.8.0",
     "eslint-plugin-jsx-a11y": "6.0.2",
     "eslint-plugin-react": "7.5.1",
-    "git-hooks": "1.1.9",
     "gulp": "3.9.1",
     "mocha": "4.0.1",
     "moment": "2.17.1",
     "nock": "9.2.3",
     "nsp": "2.8.1",
+    "pre-commit": "1.2.2",
     "semver": "5.3.0",
+    "shelljs": "0.8.1",
     "simple-git": "1.65.0",
     "supertest": "3.0.0"
   },
   "scripts": {
+    "build": "./node_modules/babel-cli/bin/babel.js src --out-dir dist",
```

Comment:
Will we have to launch the "build" command every time we change the code in the development process?